pkgs/applications/terminal-emulators/coreterminal/libcprime/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchzip
+, libnotify
+, cmake
+, ninja
+, full
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libcprime";
+  version = "4.2.2";
+
+  src = fetchzip {
+    url = "https://gitlab.com/cubocore/${pname}/-/archive/v${version}/${pname}-v${version}.tar.gz";
+    sha256 = "sha256-RywvFATA/+fDP/TR5QRWaJlDgy3EID//iVmrJcj3GXI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  buildInputs = [
+    full
+    libnotify
+  ];
+
+  meta = with lib; {
+    description = "A library for bookmarking, saving recent activites, managing settings of C-Suite";
+    homepage = "https://gitlab.com/cubocore/coreapps/libcprime";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ dan4ik605743 ];
+    platforms = platforms.linux;
+  };
+}
```

Comment:
Could you move this expression to libraries section. This library also can be used for packaging other cubocore applications.

Reply:
Yes of course